### PR TITLE
use regex match to check if message starts with command

### LINKED ISSUE #43: Bot responds to single ! or !!
React only to !word , ignore others

### PR DESCRIPTION
**Pull request by ancho**
Fixes #43

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -140,6 +140,10 @@ def load_settings(self, data):
             await self.send_text(room,
                                  f"Sorry. I don't know what to do. Execute !help to get a list of available commands.")
 
+    @staticmethod
+    def starts_with_command(body):
+        return re.match(r"^!\w.*", body)
```
Comment (vranki) [inline bot.py]: This could return boolean to be more clear and avoid testing None. Also add comment "Checks if body starts with ! and has one or more letters after it" as regexps can be difficult to read.

**Comment by ancho** [comment]
hmmm.....am i crazy? that does not work as expected....evaluates always to false and stops

**Comment by vranki** [comment]
I assume this has been tested before PR..

**Comment by ancho** [comment]
I did. It works. It was the matrix server I'm connected with, which currently sends random shadow messages to my test room and has awfull delays receiving messages. It works. Nevermind.

**Comment by vranki** [comment]
Ack, feel free to merge. LGTM.
